crates/re_renderer/shader/utils/sphere_quad.wgsl

```diff
@@ -110,3 +110,13 @@ fn sphere_quad_coverage(world_position: Vec3, radius: f32, point_center: Vec3) -
     // Note that we have signed distances to the sphere surface.
     return saturate(0.5 - distance_to_surface_in_pixels);
 }
+
+/// Computes coverage of a sphere or circle on a quad.
+///
+/// Note that in orthographic mode, spheres are always circles.
+fn circle_quad_coverage(world_position: Vec3, radius: f32, point_center: Vec3) -> f32 {
+    let to_center = point_center - world_position;
+    let distance = length(to_center);
+    let distance_pixel_difference = fwidth(distance);
```

Comment:
Drive-by comment: Seems to crash in the [web build](https://app.rerun.io/commit/3c7ee6a/index.html) (not sure if I did something wrong)

```
panicked at 'wgpu error: Validation Error

Caused by:
    In Device::create_render_pipeline
    Internal error in ShaderStages(NONE | VERTEX) shader: ERROR: 0:362: 'fwidth' : no matching overloaded function found
```

Reply:
Good catch - it's because of this bug:
* https://github.com/gfx-rs/naga/issues/1743

The work-around it to split the fragment and vertex shaders into different files, same as for `rectangle.wgsl`